error for 'make all_enrich' random.SystemRandom.choice

seeing this error today afternoon and per Chris pinging @hrshdhgd

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/.venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/.venv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/.venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/.venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/src/ontogpt/cli.py", line 728, in eval_enrichment
    print(f"RANDOM GENE: {eval_engine.random_gene_symbol()}")
  File "/Users/marcin/Documents/VIMSS/ontology/LLMs/ontogpt/src/ontogpt/evaluation/enrichment/eval_enrichment.py", line 189, in random_gene_symbol
    ann = random.SystemRandom.choice(assocs)
TypeError: choice() missing 1 required positional argument: 'seq'
make: *** [analysis/enrichment/EDS-results-2.yaml] Error 1

--- a/src/ontogpt/evaluation/enrichment/eval_enrichment.py
+++ b/src/ontogpt/evaluation/enrichment/eval_enrichment.py
@@ -186,7 +186,7 @@ class EvalEnrichment(EvaluationEngine):
         """Get a random gene."""
         assocs = list(self.ontology.associations())
         logger.debug(f"Got {len(assocs)} associations")
-        ann = random.SystemRandom.choice(assocs)
+        ann = random.SystemRandom().choice(assocs)
         info = gene_info(ann.subject)
         return info[0]
 

